### PR TITLE
agent: Add some mount options and sort the options alphabetically

### PR DESCRIPTION
**Pull request by ManaSugi**
Add the following mount options to catch up with the runtime spec
- `silent`
- `loud`
- `(no)acl`
- `(no)iversion`
- `(no)lazytime`

Fixes: #1999 

Signed-off-by: Manabu Sugimoto <Manabu.Sugimoto@sony.com>

### CODE REVIEW

**Review by liubin — APPROVED** [review]

**Review by snir911 — APPROVED** [review]
LGTM

File: src/agent/rustjail/src/mount.rs [hunk 1]
```diff
@@ -97,8 +97,16 @@ lazy_static! {
...
+        m.insert("silent", (false, MsFlags::MS_SILENT));
+        m.insert("loud", (true, MsFlags::MS_SILENT));
+        m.insert("acl", (false, MsFlags::MS_POSIXACL));
+        m.insert("noacl", (true, MsFlags::MS_POSIXACL));
+        m.insert("iversion", (false, MsFlags::MS_I_VERSION));
+        m.insert("noiversion", (true, MsFlags::MS_I_VERSION));
         m.insert("strictatime", (false, MsFlags::MS_STRICTATIME));
         m.insert("nostrictatime", (true, MsFlags::MS_STRICTATIME));
+        m.insert("lazytime", (false, MsFlags::MS_LAZYTIME));
+        m.insert("nolazytime", (true, MsFlags::MS_LAZYTIME));
```
Comment (jodh-intel) [inline src/agent/rustjail/src/mount.rs:109]: Not strictly required for this PR, but you might like to add a second commit to sort `PROPAGATION` and `OPTIONS` alphabetically. Even though a number of these entries are effectively in pairs ("`foo`, "`nofoo`"), it's hard to visually scan over the list currently.

**Review by jodh-intel — APPROVED** [review]
Thanks @ManaSugi.

lgtm

File: src/agent/rustjail/src/mount.rs (hunk 1 above)
Comment (ManaSugi) [reply src/agent/rustjail/src/mount.rs]: Thank you for the comment.
You're completely right .
I created the second commit and added the description to the current issue #1999.

Comment (jodh-intel) [reply src/agent/rustjail/src/mount.rs]: Thanks very much @ManaSugi !

**Review by snir911 — APPROVED** [review]
LGTM with the sorting as well :) thanks!

**Review by liubin — APPROVED** [review]

**Pull request merged**